$MirOS: ports/misc/mc/patches/patch-vfs_extfs_sfs_ini,v 1.1 2008/09/17 20:07:00 tg Exp $
--- vfs/extfs/sfs.ini.orig	Tue Dec 15 15:57:22 1998
+++ vfs/extfs/sfs.ini	Fri Dec 31 17:44:06 2010
@@ -10,6 +10,10 @@ bz/1	bzip < %1 > %3
 ubz/1	bzip -d < %1 > %3
 bz2/1	bzip2 < %1 > %3
 ubz2/1	bzip2 -d < %1 > %3
+lzma/1	lzma <%1 >%3
+ulzma/1	lzmadec <%1 >%3
+xz/1	xz <%1 >%3
+uxz/1	xzdec <%1 >%3
 tar/1	tar cf %3 %1
 tgz/1	tar czf %3 %1
 uhtml/1	lynx -force_html -dump %1 > %3
